### PR TITLE
Update lock files after clearing cache

### PR DESCRIPTION
**Pull request by gaya3-zipstack**
## What

- Fix wrong versions of dependent llama-index libs

## Why

- lock files were having outdated versions of llama-index adapter lib even though llama-index lib was updated to 0.10.38

## How

- Clear pdm cache and re-generate lock file

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

- No

## Database Migrations

- NA

## Env Config

- NA

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

- Did  pdm run backend and could not notice any import error as mentioned in https://zipstack.slack.com/archives/C06SNT0FTU5/p1718620413856559?thread_ts=1718619074.231729&cid=C06SNT0FTU5

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

**Comment by Deepak-Kesavan** [comment]
@gaya3-zipstack Looks like you have used the old branch which was merged earlier to main. It is creating merge conflict with main and showing old diff which was already in main.

Also, SDK patch version needs to be updated for this right?

File: src/unstract/sdk/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__version__ = "0.31.1"
+__version__ = "0.32.0"
```
Comment (chandrasekharan-zipstack) [inline src/unstract/sdk/__init__.py:1]:
```suggestion
__version__ = "0.31.2"
```
Please fix the conflicts on your branch as well

**Review by chandrasekharan-zipstack — APPROVED** [review]

**Review by hari-kuriakose — APPROVED** [review]
@gaya3-zipstack Looks good overall.

File: src/unstract/sdk/__init__.py (hunk 1 above)
Comment (gaya3-zipstack) [reply src/unstract/sdk/__init__.py]: Done. Versin in 0.32.0

**Pull request merged**